ui/src/layout/Assets/AssetsTable.js

```diff
@@ -67,7 +67,7 @@ const AssetsTable = () => {
             Header: "Location",
             id: "location",
             sortIds: LOCATION_SORT_IDS,
-            accessor: "assetInfo.location"
+            accessor: (original) => original.assetInfo.location || original.assetInfo.repoDigests?.[0] || original.assetInfo.image?.repoDigests?.[0]
```

Comment:
I'm not sure about using the image repoDigest as the location for a Container that doesn't seem like the right approach. I think ideally if assetInfo.location doesn't exist, then we should put "unknown" or something. Once the multi-cloud stuff merges I think we'll want to replace all of this with the provider(s) which found the asset. @paralta   is adding a note about this to the multi-cloud design doc.

Reply:
@gallotamas Could you pls update the PR with @Tehsmash suggestion. I have overlooked this part. :/

Reply:
sure, the location will be empty in this case since we don't get the data from the API but you can populate it with the right information and then it will be displayed properly 🙂